test/multivariate/mvlogitnormal.jl

```diff
@@ -0,0 +1,163 @@
+# Tests on Multivariate Logit-Normal distributions
+using Distributions
+using ForwardDiff
+using LinearAlgebra
+using Random
+using Test
+
+####### Core testing procedure
+
+function test_mvlogitnormal(d::MvLogitNormal; nsamples::Int=10^6)
+    @test d.normal isa AbstractMvNormal
+    dnorm = d.normal
+
+    @testset "properties" begin
+        @test length(d) == length(dnorm) + 1
+        @test params(d) == params(dnorm)
+        @test partype(d) == partype(dnorm)
+        @test eltype(d) == eltype(dnorm)
```

Comment:
With the suggestion above, we should test both the type and instance based definition:
```suggestion
        @test eltype(d) == eltype(dnorm)
        @test eltype(typeof(d)) == eltype(typeof(dnorm))
```